drake/systems/DrakeSystem.cpp

```diff
@@ -63,12 +104,12 @@ inline void handle_realtime_factor(const TimePoint& wall_clock_start_time, doubl
 }
 
 
-void DrakeSystem::ode1(double t0, double tf, const DrakeSystem::VectorXs& x0, const SimulationOptions& options) const {
+void DrakeSystem::ode1(double t0, double tf, const VectorXd& x0, const SimulationOptions& options) const {
```

Comment:
It doesn't need to be a reference.  The benefit would be if someone accidentally modifies it in the function body it would result in a compile time error.